cylc/flow/task_pool.py

```diff
@@ -597,10 +597,6 @@ def release_runahead_task(
             # implicit prev-instance parent
             return
 
-        if not itask.flow_nums:
-            # No reflow
-            return
```

Comment:
@hjoliver is this ok?

Reply:
You're on the right track, but this doesn't fix the problem.